extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -30,9 +33,26 @@ define('ext.wikia.adEngine.video.googleIma', [
 		return adsRequest;
 	}
 
+	function createProgressBar() {
+		var progressBar = doc.createElement('div'),
+			currentTime = doc.createElement('div');
+
+		progressBar.classList.add('progress-bar');
+		currentTime.classList.add('current-time');
+
+		progressBar.appendChild(currentTime);
+
+		return progressBar;
+	}
+
 	function prepareVideoAdContainer(videoAdContainer) {
+		var progressBar = createProgressBar();
+
 		videoAdContainer.style.position = 'relative';
 		videoAdContainer.classList.add('hidden');
+		videoAdContainer.classList.add('video-ima-container');
```

Comment:
According to e-mail from 10/13/2016 we stopped actively supporting IE :smiling_imp: And Edge supports it, so how about writing it down in known issues and keeping the code as clean as possible?